reference/rpc/sendtypeto.rst

```diff
@@ -60,19 +66,24 @@ Argument #7 - coin_control
 **Type:** json, optional
 
 Coincontrol object.
-       7.1 "changeaddress"  (string, optional) The Address for receiving change
-       7.2 "inputs"         (json, optional) 
-       [{"tx":, "n":},...],
-       7.3 "replaceable"    (boolean, optional)  Allow this transaction to be replaced by a transaction
-       with higher fees via BIP 125
-       7.4 "conf_target"    (numeric, optional) Confirmation target (in blocks)
-       7.5 "estimate_mode"  (string, optional) The fee estimate mode, must be one of:
-       "UNSET"
-       "ECONOMICAL"
-       "CONSERVATIVE"
-       7.6 "fee_rate"        (numeric, optional, default not set: makes wallet determine the fee) Set a specific 
-       feerate (UTE per KB)
-       7.7 "ignore_remote_staked" (boolean, opional, default=false) Exclude coins that are currently staked on other nodes.
+
+::
+
+  {
+    "changeaddress": "<address>", (string, optional) The Address for receiving change
+    "inputs":                     (json, optional)
+           [{"tx":, "n":},...],
```

Comment:
Since we're touching this anyway, could we maybe add a description for this argument too? Something like the `createrawtransaction` documentation:
```
  [{
    "tx": "id",     (string, required) The transaction id
    "n": n,         (numeric, required) The output number
  }, ... ]
```
Same with `stakeat`.

Reply:
Makes sense to me. This pull request is not touching the actual documentation, though, but just the rendering of what is in the code. The pull request changing the messages was https://github.com/dtr-org/unit-e/pull/1030. We had a discussion there that we would postpone bigger cleanups of the docs to after the upstream sync because the framework has changed there a bit.

Reply:
Let's wait for the merge, then.